app/src/test/java/de/westnordost/streetcomplete/quests/opening_hours_signed/CheckOpeningHoursSignedTest.kt

```diff
@@ -85,4 +133,29 @@ class CheckOpeningHoursSignedTest {
             StringMapEntryModify("check_date:opening_hours", "2020-03-04", LocalDate.now().toCheckDateString()),
         )
     }
+    
+    @Test fun `apply no answer with existing opening hours via other means`() {
+        questType.verifyAnswer(
+            mapOf(
+                "opening_hours" to "24/7",
+                "opening_hours:signed" to "no"
+            ),
+            false,
+            StringMapEntryModify("opening_hours:signed", "no", "no"),
+            StringMapEntryAdd("check_date:opening_hours", LocalDate.now().toCheckDateString()),
+        )
+    }
+
+    @Test fun `apply no answer with prior check date and existing opening hours via other means`() {
+        questType.verifyAnswer(
+            mapOf(
+                "opening_hours" to "Mo 10:00-12:00",
+                "opening_hours:signed" to "no",
+                "check_date:opening_hours" to "2020-03-04"
+            ),
+            false,
+            StringMapEntryModify("opening_hours:signed", "no", "no"),
+            StringMapEntryModify("check_date:opening_hours", "2020-03-04", LocalDate.now().toCheckDateString()),
```

Comment:
this also seems weird, and ideally would be mentioned why it is not using `check_date:opening_hours:signed` (but can be treated as out of scope of that PR)